projects/plugins/crm/includes/ZeroBSCRM.DAL3.Helpers.php

```diff
@@ -5696,6 +5696,35 @@ function zeroBSCRM_check_amount_due_mark_paid($invoice_id=-1){
 		return false;
 	}
 
+/**
+ * Helper function calculate the number of deleted invoices or transactions and the
+ * total value of those invoices, for any particular contact / company.
+ *
+ * @param array $all_invoices An array of all invoice or transaction data for a contact / company.
+ *
+ * @returns array An array with the deleted invoices total, and the deleted invoices count.
+ */
+function jetpackCRM_deleted_totals( $all_invoices = null ) {
```

Comment:
Typically we've prefixed new functions with `jpcrm_` (eventually namespace will remove the need for a prefix), but not a blocker. However:

* the name of the function here isn't very descriptive as to what it does/what it returns
* the `$all_invoices` param and the logic throughout indicates it only handles invoices

It seems we've expanded some of the logic to transactions as well as invoices. For better or worse, transaction counts/statuses are manually controlled in the settings and [filtered a level up in the query](https://github.com/Automattic/jetpack/blob/0835a5018bcc12737ed94ba66fdae6960496417a/projects/plugins/crm/includes/ZeroBSCRM.DAL3.Obj.Contacts.php#L1861) (if I'm reading the code correctly, and [here's companies](https://github.com/Automattic/jetpack/blob/0835a5018bcc12737ed94ba66fdae6960496417a/projects/plugins/crm/includes/ZeroBSCRM.DAL3.Obj.Companies.php#L468)). As such, won't this only affect invoices?

If correct, I'd propose reducing the scope of the function and calling it something like:

`jpcrm_deleted_invoice_total_and_counts()`

Reply:
I've now changed the scope back to transactions so the params and logic will be ok, but I've changed the function name, thanks!